presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentConfig.java

```diff
@@ -46,8 +45,13 @@ default String getConfigName()
         return nameForConfigClass(getClass());
     }
 
-    default Optional<EnvironmentExtender> extendEnvironment(String environmentName)
+    default EnvironmentExtender getEnvironmentExtender(String environmentName)
```

Comment:
Why do we need `environmentName` here?

Reply:
Because config applies to multiple environments and sometimes there is a need for config to configure only selected environment(s). `EnvironmentExtender` accepts `Environment.Builder` which does not expose environment name.